src/instructlab/configuration.py

```diff
@@ -661,6 +665,15 @@ class _train(BaseModel):
     )
 
 
+class _prompts(BaseModel):
```

Comment:
is this used anywhere? I don't see this being use directly in a cmd so I am hesitant to add a field without the usecase

Reply:
Please correct me if I am misunderstanding, here is what I check:
1. Currently, the `SYS_PROMPT` is using in `chat` from the handcode `common.py`
2. Due to different models, or different situations for the user, so the prompt might be need to modify by user frequently.
3. Change from here: define a default one from `common.py`  -->     write into `config.yaml` with init --> read the prompt from the config for `chat` --> user can use `ilab config edit` to modify the prompt

Reply:
@cdoern can you respond to the above please

Reply:
thanks for response.  I do see a user ask for change(system prompt) in `users` channel, may consider set it in config.